doc/level1.rst

```diff
@@ -1142,6 +1142,27 @@ looking for a Setup Message again.  A more sophisticated
 implementation may offer a more user-friendly way to detect this
 situation and resolve it.
 
+Cold E-mail
+-----------
+
+tbd: this is why cold e-mail is cool
+
+
+mailto: URI
++++++++++++
+
+The following query parameter are used by Autocrypt:
+
+ * `autocrypt-keydata`: The Base64 representation of the binary :rfc:`OpenPGP "Transferable Public Key"<4880#section-11.1>` of the address the mailto URI is referring to, as specified for the Autocrypt header without newlines and with any padding characters percent-encoded.
+ * `autocrypt-prefer-encrypt`: The prefer-encrypt value (either nopreference or mutual) of the address, as specified for the Autocrypt header. 
+
+The values derived from the mailto URI MUST not modify the existing Autocrypt Peer State for the given address. The values MAY only be used for composing the e-mail that was invoked by the mailto URI. If the local Autocrypt Peer State has a key for the given address and its `preliminary-recommendation` is calculated to be `available` (meaning not outdated and not received through gossip), that key MUST take precedent over the one supplied through the `mailto` URI.
```

Comment:
This could be formatted with https://sembr.org/ for better readability and better diffs?